code/datums/martial/plasma_fist.dm

```diff
@@ -6,20 +6,28 @@
 	name = "Plasma Fist"
 	id = MARTIALART_PLASMAFIST
 	help_verb = /mob/living/carbon/human/proc/plasma_fist_help
+	var/plasma_power = 1
 
 
 /datum/martial_art/plasma_fist/proc/check_streak(mob/living/carbon/human/A, mob/living/carbon/human/D)
 	if(findtext(streak,TORNADO_COMBO))
+		if(A == D)//helps using apotheosis
+			return 0
```

Comment:
```suggestion
			return FALSE
```

Reply:
agreed and i should turn all the rest of the 1/0's in this file to TRUE/FALSE as well